libs/core/src/lib/switch/Switch.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Pressable, View, StyleSheet } from 'react-native';
+
+import { Palette } from '@bmg-react-native-nx/constants';
+
+type SwitchProps = {
+  on: boolean;
+  onPress?: () => void;
+};
+
+export function Switch({ on, onPress }: SwitchProps) {
+  const [isEnabled, setIsEnabled] = useState(on);
+
+  const toggleSwitch = () => {
+    setIsEnabled((previousState) => !previousState);
+  };
```

Comment:
აქედან დაათრიგერე `onChange` ივენთი

```suggestion
  const toggleSwitch = () => {
    setIsEnabled((previousState) => !previousState);
    onChange(isEnabled);
  };
```